tests/test_case_lib.py

```diff
@@ -1,234 +1,199 @@
+    with s3:
 
-            # test init with cidr_range
-            case_with_cidr = case.Case(
-                examiner_cidr_range='8.8.8.8/32'
-            )
-            self.created_buckets.append(case_with_cidr.case_bucket)
-
-            assert case_with_cidr is not None
-            assert case_with_cidr.case_number is not None
-            assert case_with_cidr.case_bucket is not None
-            assert case_with_cidr.examiner_cidr_range == '8.8.8.8/32'
-
-    def test_rename_log_file(self):
-        # remove this test once we can test case.teardown
-        self.generic_case = case.Case()
-        self.log_base = os.path.dirname(os.path.abspath(__file__))
-
-        self.test_log = "{0}/{1}-aws_ir.log".format(self.log_base,
-                                                    self.generic_case.case_number)
-        self.renamed_test_log = "{0}/{1}-{2}-aws_ir.log".format(
-            self.log_base,
-            self.generic_case.case_number,
-            'i-12345678'
+        s3_resource.Bucket(existing_bucket_name).create()
+        created_buckets.append(existing_bucket_name)
+
+        case_with_bucket = case.Case(
+            case_bucket=existing_bucket_name
         )
 
-        with open(self.test_log, 'w') as f:
-            f.write('test log data')
-            f.close()
+        assert case_with_bucket is not None
+        assert case_with_bucket.case_number is not None
+        assert case_with_bucket.case_bucket == existing_bucket_name
+        assert case_with_bucket.examiner_cidr_range == '0.0.0.0/0'
 
-        # create test screenshot
-        self.test_jpg = "{0}/{1}-console.jpg".format(
-            self.log_base,
-            self.generic_case.case_number
-        )
 
-        with open(self.test_jpg, 'w') as f:
-            f.write('test jpg data')
-            f.close()
+def test_rename_log_file():
+    # remove this test once we can test case.teardown
+    generic_case = case.Case()
 
-        result = self.generic_case._rename_log_file(
-            self.generic_case.case_number,
-            'i-12345678',
-            base_dir=self.log_base
-        )
+    test_log = "{0}/{1}-aws_ir.log".format(log_base, generic_case.case_number)
 
-        print(result)
+    with open(test_log, 'w') as f:
+        f.write('test log data')
+        f.close()
 
-        assert result is True
+    # create test screenshot
+    test_jpg = "{0}/{1}-console.jpg".format(
+        log_base,
+        generic_case.case_number
+    )
 
-    def test_copy_logs_to_s3(self):
-        # setup bucket to test providing an s3 bucket to Case class
-        self.m = mock_s3()
+    with open(test_jpg, 'w') as f:
+        f.write('test jpg data')
+        f.close()
 
-        self.m.start()
+    result = generic_case._rename_log_file(
+        generic_case.case_number,
+        'i-12345678',
+        base_dir=log_base
+    )
 
-        self.e = mock_ec2()
-        self.e.start()
+    print(result)
```

Comment:
print statement in code.  Remove or replace with logger.  Maybe this was for test?